src/backend/access/transam/xlog.c

```diff
@@ -7505,8 +7506,17 @@ StartupXLOG(void)
 				 * xlogreader in this function.
 				 */
 				if (record->xl_rmid == RM_XLOG_ID)
+				{
 					VerifyOverwriteContrecord(record, xlogreader);
 
+					/*
+					 * TODO: add call to PdlRedoDropFiles() when
+					 * RemovePendingDeletesForPreparedTransactions() is
+					 * implemented. It should be called right before PdlRedoDropFiles().
+					 * Without it test 'crash_recovery_dtm' fails.
+					 */
```

Comment:
We use tracker to plan our tasks and never find comments with TODO. So this comment and the same one below are meaningless. I suggest to remove them and write this text to some existing ticket or create a new one.